enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -128,9 +130,17 @@ where
 
 	// state_get
 	let state_get_name: &str = "state_get";
-	io.add_sync_method(state_get_name, |_: Params| {
-		let parsed = "world";
-		Ok(Value::String(format!("hello, {}", parsed)))
+	io.add_sync_method(state_get_name, move |params: Params| {
+		let json_value = match execute_getter_inner(getter_executor.as_ref(), params) {
+			Ok(state_getter_value) => RpcReturnValue {
+				do_watch: false,
+				value: state_getter_value.encode(),
+				status: DirectRequestStatus::Ok,
```

Comment:
That's a breaking change. Could you please mention this in the PR description?
E.g. 
TrustedGetter now returns
```rust
RpcReturnValue {
	do_watch: false,
value: state_getter_value.encode(),
status: DirectRequestStatus::Ok,}
```

instead of 
```rust
RpcReturnValue {
	do_watch: false,
      value: state_getter_value.encode(),
      status: DirectRequestStatus::TrustedOperationStatus::Submitted
,}

```

Reply:
Yes 👍  done.